src/main.rs

```diff
@@ -1,144 +1,208 @@
-
-            crate::app::register_server_functions();
-
-            // Setting this to None means we'll be using cargo-leptos and its env vars.
-            // when not using cargo-leptos None must be replaced with Some("Cargo.toml")
 
-            let conf = leptos::get_configuration(Some("Cargo.toml")).await.unwrap();
-
-            let addr = conf.leptos_options.site_addr.clone();
-            let routes = generate_route_list(|cx| view! { cx,
-               <App/> });
+cfg_if! { if #[cfg(feature = "ssr")] {
+    use leptos::*;
+    use crate::app::*;
+    //mod server_actions;
+    mod backend;
+    use backend::Backend;
+    extern crate ord_mini;
+    use ord_mini::Inscription;
+    use std::collections::HashMap;
+    extern crate dotenv;
+    extern crate num_format;
+    extern crate leptos_axum;
+    use leptos_axum::{generate_route_list, LeptosRoutes};
+    mod fallback;
+
+    use axum::body::Body as AxumBody;
+    use axum::{
+        extract::{Extension, Path},
+        http::Request,
+        response::{IntoResponse, Response},
+        routing::{get, post},
+        Router,
+    };
+
+    pub mod error_template;
+    pub mod errors;
+
+    mod server_actions;
+
+    use axum::{
+        extract::TypedHeader,
+        response::sse::{Event, Sse},
+    };
+    use futures::stream::{self, Stream};
+    use std::{convert::Infallible, time::Duration};
+    use tokio_stream::StreamExt as _;
+    use tower_http::trace::TraceLayer;
+    use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
+    use std::sync::Arc;
+
+}}
+mod app;
 
-            let mut ordipool:  HashMap<String, Option<Inscription>> = HashMap::new();
-            let backend = std::env::var("BACKEND").unwrap_or("bitcoin_core".to_string()).to_lowercase();
-            //let backend_str = backend.as_str();
-            let backend_space = backend::Space::new();
-            let backend_bitcoin_core = backend::BitcoinCore::new();
+//Define a handler to test extractor with state
+#[cfg(feature = "ssr")]
+async fn custom_handler(
+    Path(id): Path<String>,
+    Extension(options): Extension<Arc<LeptosOptions>>,
+    req: Request<AxumBody>,
+) -> Response {
+    let handler = leptos_axum::render_app_to_stream_with_context(
+        (*options).clone(),
+        move |cx| {
+            provide_context(cx, id.clone());
+        },
+        |cx| view! { cx, <App/> },
+    );
+    handler(req).await.into_response()
+}
 
-            // todo print more relevant config stuff
-            dbg!(&backend_bitcoin_core);
-            if std::env::var("RUST_BACKTRACE").is_ok() {
-                let mut buffer = String::new();
-                let stdin = std::io::stdin(); // We get `Stdin` here.
-                println!("Press enter to start with config above. Crtl+C to abort.");
-                stdin.read_line(&mut buffer)?;
+// #[get("/style.css")]
+// async fn css() -> impl Responder {
+//     actix_files::NamedFile::open_async("./style/output.css").await
+// }
```

Comment:
Yep, that's an Actix artifact and can be removed - `css` is already linked in `App`.